wagtailio/showcase/models.py

```diff
@@ -0,0 +1,176 @@
+from django.core.exceptions import ValidationError
+from django.core.paginator import EmptyPage, PageNotAnInteger, Paginator
+from django.db import models
+
+from modelcluster.fields import ParentalKey
+from wagtail.admin.panels import FieldPanel, InlinePanel, MultiFieldPanel
+from wagtail.fields import StreamField
+from wagtail.models import Orderable, Page
+from wagtail.search import index
+
+from wagtailio.core.blocks import StandaloneCTABlock
+from wagtailio.showcase.snippets import Sector
+from wagtailio.utils.models import CrossPageMixin, SocialMediaMixin
+
+
+class ShowcasePage(SocialMediaMixin, CrossPageMixin, Page):
+    template = "patterns/pages/showcase_page/showcase_page.html"
+    htmx_template = "patterns/pages/showcase_page/showcase_page_fragment.html"
+
+    parent_page_types = ["core.HomePage"]
+
+    introduction = models.TextField(
```

Comment:
I still think this should be a `RichTextField` with `p, a, ul, ol, link, document-link`

Reply:
Fair, sorry, I didn't mean to ignore this!.